app/components/Project/ProjectForm.tsx

```diff
@@ -99,38 +87,28 @@ const ProjectForm: React.FC<Props> = (props) => {
         operatorTradeName: {
           type: "string",
         },
-        fundingStreamId: {
-          type: "number",
-          title: "Funding Stream ID",
-          default: undefined,
-          anyOf: query.allFundingStreams.edges.map(({ node }) => {
-            return {
-              type: "number",
-              title: node.name,
-              enum: [node.rowId],
-              value: node.rowId,
-            };
-          }),
-        },
-        fundingStreamDescription: {
-          type: "string",
-        },
         fundingStreamRfpId: {
           type: "number",
           title: "Funding Stream RFP ID",
           default: undefined,
-          anyOf: rfpOptions.map(({ node }) => {
-            return {
-              type: "number",
-              title: node.year.toString(),
-              enum: [node.rowId],
-              value: node.rowId,
-            };
-          }),
+          parent: {
+            entityType: "allFundingStreams",
+            displayField: "name",
+            placeholder: "Select a Funding Stream",
+            label: "Funding Stream"
+          },
+          child: {
+            entityType: "allFundingStreamRfps",
```

Comment:
One problem here is that there is a relation between the parent and child (not all funding streams exist on a given year), but it is not expressed here